oci/casext/gc.go

```diff
@@ -35,7 +38,7 @@ import (
 // functions. In other words, it assumes it is the only user of the image that
 // is making modifications. Things will not go well if this assumption is
 // challenged.
```

Comment:
This doc comment needs to be updated.

Reply:
That the blob list is no longer "stable" while making the GC() call? Still true in our use case as we hold a lock when calling GC().

Reply:
I mean that you need to update the doc comment to explain what the `policies` arguments do.